components/switch/README.md

```diff
@@ -49,7 +49,6 @@ The `d2l-switch` element is a generic switch with on/off semantics.
 | `disabled` | Boolean | Disables the switch |
 | `on` | Boolean | Whether the switch is "on" or "off" |
 | `text-position` | String | Valid values are: `start`, `end` (default), and `hidden` |
-| `tooltip` | String | Text to display in a tooltip for the switch |
```

Comment:
Note that the changes in this file and several others below where there is just stuff being deleted are actually from [this PR](https://github.com/BrightspaceUI/core/pull/2485), and it got added here probably because I switched between the `feature branch` and `main` as the base branch of the PR.